### DISCUSSION TITLE
Random Questions

### DISCUSSION THREAD

**Discussion by ghost** [winstxnhdw/lc-hax#493] [General]
I'm not asking you to implement these features, but do you think it's possible and how would someone do this?
- Spamming the server list with fake servers
- Crashing a player's game without crashing everyone
- Take away god mode from another hacker

Thanks :D

**Comment by winstxnhdw** [comment]
Oh, this is a great question.

- Spamming the server list

This can either be really hard or really easy but if I were to do it, I’d start looking at methods for hosting a match. It’s likely going to be using the Steam API so I’ll ignore every Unity.Netcode method. I’ll try spamming the Steam API method responsible for hosting and see if something happens. But it’s likely the Steam API will need some kind of lifespan/dependency. Looking at their documentation may help.


- Crashing a player’s game without crashing anyone

This isn’t possible. Everyone’s network is dependent on the host, so if you crash anyone, you inadvertently crash the host, and therefore, you crash everyone.

- Take god mod away from another player

This is also not really feasible. At best, you can see how their god mode works and see if any other RPC bypasses that. For example, `lc-hax` has the `fatality` command which usually bypasses God mode.

**Comment by AzazelHD** [comment]
- Is there a way to disable the hack once started without closing the game?
- Is there a way to disable the no weight hack?
- Is there a way to turn off night vision?

**Reply by winstxnhdw** [reply]
> Is there a way to disable the hack once started without closing the game?

Not at the moment. As for the rest, read the docs.

**Reply by AzazelHD** [reply]
Ive read the docs, but i dont see you can do it, thats why i ask. You can disable them by commenting the respective line, but not like turn them on/off.

**Reply by winstxnhdw** [reply]
So you just got your answer. It can't be done.

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/ecf7055d-6a0c-44da-828d-ef5777d93f65)
